src/app/components/shell/cbar/community/community.vue

```diff
@@ -1,69 +1,89 @@
 <script lang="ts" src="./community"></script>
 
 <template>
-	<app-popper
-		popover-class="fill-darkest"
-		trigger="right-click"
-		placement="right"
-		fixed
-		block
-		hide-on-state-change
-		@show="popperVisible = true"
-		@hide="popperVisible = false"
-	>
-		<template #default>
-			<div @click.capture="onCommunityClick">
-				<app-shell-cbar-item
-					class="-community"
-					:is-active="isActive"
-					:is-unread="isUnread"
-					:highlight="highlight"
-				>
-					<router-link
-						v-app-tooltip.right="tooltip"
-						class="-link link-unstyled"
-						:to="{
-							name: 'communities.view.overview',
-							params: { path: community.path },
-						}"
+	<div>
+		<app-popper
+			popover-class="fill-darkest"
+			trigger="right-click"
+			placement="right"
+			fixed
+			block
+			hide-on-state-change
+			@show="popperVisible = true"
+			@hide="popperVisible = false"
+		>
+			<template #default>
+				<div @click.capture="onCommunityClick">
+					<app-shell-cbar-item
+						class="-community"
+						:is-active="isActive"
+						:is-unread="isUnread"
+						:highlight="highlight"
 					>
-						<app-media-item-backdrop
-							class="-backdrop"
-							:media-item="community.thumbnail"
-							radius="full"
+						<router-link
+							v-app-tooltip.right="tooltip"
+							class="-link link-unstyled"
+							:to="{
+								name: 'communities.view.overview',
+								params: { path: community.path },
+							}"
 						>
-							<app-community-thumbnail-img class="-thumb" :community="community" />
-						</app-media-item-backdrop>
-					</router-link>
-				</app-shell-cbar-item>
+							<app-media-item-backdrop
+								class="-backdrop"
+								:media-item="community.thumbnail"
+								radius="full"
+							>
+								<app-community-thumbnail-img
+									class="-thumb"
+									:community="community"
+								/>
+							</app-media-item-backdrop>
+						</router-link>
+					</app-shell-cbar-item>
+				</div>
+			</template>
 
-				<hr v-if="!community.is_member" class="-hr" />
-			</div>
-		</template>
+			<template #popover>
+				<div class="list-group list-group-dark">
+					<app-community-perms :community="community">
+						<router-link
+							class="list-group-item has-icon"
+							:to="community.routeEditLocation"
+						>
+							<app-jolticon icon="edit" />
+							<translate>Edit Community</translate>
+						</router-link>
+					</app-community-perms>
+					<a
+						v-if="shouldShowLeave"
+						class="list-group-item has-icon"
+						@click="onLeaveCommunityClick"
+					>
+						<app-jolticon icon="remove" notice />
+						<translate>Leave Community</translate>
+					</a>
+					<a
+						v-else-if="shouldShowJoin"
+						class="list-group-item has-icon"
+						@click="onJoinCommunityClick"
+					>
+						<app-jolticon icon="add" notice />
+						<translate>Join Community</translate>
+					</a>
+					<a
+						v-if="shouldShowModerate"
```

Comment:
This is currently showing both `Join` and `Moderate` for communities in which you have collaborator status. I'm not sure if we need to show both at any point, but fixing the previous issue in the TS file should resolve this just fine.